miner/worker.go

```diff
@@ -1026,6 +1154,13 @@ func (w *worker) commitNewWork(interrupt *int32, noempty bool, timestamp int64)
 			localTxs[account] = txs
 		}
 	}
+	var cnt int64
+	if len(l1messages) > 0 {
+		cnt = w.commitL1Messages(l1messages, w.coinbase, interrupt)
```

Comment:
Can't we just use `commitTransactions(l1messages)`? With `types.NewTx(&msg)`.

Reply:
`commitTransactions` accepts `*types.TransactionsByPriceAndNonce` which is made from `txs map[common.Address]Transactions`

`*types.TransactionsByPriceAndNonce` spec:
// TransactionsByPriceAndNonce represents a set of transactions that can return
// transactions in a profit-maximizing sorted order, while supporting removing
// entire batches of transactions for non-executable accounts.
It sorts `txs` by `price` and then by `time` if prices are equal

Reply:
Ok, I see. Times of `txs` will be increasing if use `types.NewTx` in correct order